Add a spec file template + relevant fixes

lzaoral opened the pull request:
This specfile template can be used with the `rpkg` tool, which can create a SRPM from a git repository. The resulting SRPM can be used with CentOS, Fedora, openSuse and Mageia. However, both openSuse and Mageia need some changes done to the build process of `dg`.

lzaoral commented:
The reason for installing them was that these tools are/may be executed by the wrapper scripts:

```
$ pta-show
/usr/bin/pta-show: line 20: llvm-pta-dump: command not found
```

A solution (a bit dirty though) may be to only install the wrapper scripts and `llvm-slicer` to `/usr/bin` and the rest to `/usr/libexec` and to modify the wrappers to look for the rest there.

mchalupa commented:
Well, if we wouldn't install the tools, there is no need to install the wrapper scripts too. So I would go with not installing the wrapper either.

lzaoral commented:
Ok, everything should be ready now. I have also added a notice to docs, that the packages don't provide other utilities and fixed `llvm-slicer --version`.
